recipes/libcurl/all/conanfile.py

```
@@ -134,14 +141,11 @@ def configure(self):
             raise ConanInvalidConfiguration("darwinssl only suppported on Apple like OS (Macos, iOS, watchOS or tvOS).")
         if self.options.with_ssl == "wolfssl" and self._is_using_cmake_build and tools.Version(self.version) < "7.70.0":
             raise ConanInvalidConfiguration("Before 7.70.0, libcurl has no wolfssl support for Visual Studio or \"Windows to Android cross compilation\"")
-        if self.options.with_libmetalink and tools.Version(self.version) >= "7.78.0":
```

I think we want to raise an invalid config error if a user still has `with-libmetalink` option specified and updates to >= 7.78.0 to let the user know